api/saml/provider.py

```diff
@@ -85,7 +85,11 @@ def __init__(self, library, integration, analytics=None):
                 configuration.patron_id_use_name_id
             )
             self._patron_id_attributes = (
-                json.loads(configuration.patron_id_attributes)
+                (
+                    json.loads(configuration.patron_id_attributes)
+                    if isinstance(configuration.patron_id_attributes, str)
+                    else configuration.patron_id_attributes
+                )
```

Comment:
Similar to comment on `api/odl2.py` lines +178 to +180.

Reply:
I added additional changes to https://github.com/ThePalaceProject/circulation-core/pull/35 in order to correctly parse JSON-encoded `LIST` and `MENU` items directly in `ConfigurationMetadata.__get___` and removed the code above.